app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -2,22 +2,24 @@
 
 module Authentication
   module AuthnJwt
-    PROVIDER_URI_RESOURCE_NAME = "provider-uri".freeze
-    JWKS_URI_RESOURCE_NAME = "jwks-uri".freeze
-    ISSUER_RESOURCE_NAME = "issuer".freeze
-    IDENTITY_FIELD_VARIABLE = "token-app-property".freeze
-    IDENTITY_NOT_RETRIEVED_YET = "Identity not retrieved yet".freeze
-    PRIVILEGE_AUTHENTICATE="authenticate".freeze
-    NOT_INITIALIZED_IDENTITY="".freeze
-    ISS_CLAIM_NAME = "iss".freeze
-    EXP_CLAIM_NAME = "exp".freeze
-    NBF_CLAIM_NAME = "nbf".freeze
-    IAT_CLAIM_NAME = "iat".freeze
+    PROVIDER_URI_RESOURCE_NAME = "provider-uri"
+    JWKS_URI_RESOURCE_NAME = "jwks-uri"
+    ISSUER_RESOURCE_NAME = "issuer"
+    IDENTITY_FIELD_VARIABLE = "token-app-property"
+    IDENTITY_NOT_RETRIEVED_YET = "Identity not retrieved yet"
+    PRIVILEGE_AUTHENTICATE="authenticate"
+    NOT_INITIALIZED_IDENTITY="not initialized user"
```

Comment:
Validate if we already have this functionality in code (look at the file `role_id.rb`)

Reply:
Removed